vip_hci/var/shapes.py

```diff
@@ -339,9 +339,9 @@ def get_annulus_segments(array, inner_radius, width, nsegm=8, theta_init=0,
     ----------
     array : array_like
         Input 2d array or image.
-    inner_radius : int
+    inner_radius : scalar
```

Comment:
I'd suggest to follow the convention in VIP: int or float

Reply:
changed in [commit 939fdbf](https://github.com/vortex-exoplanet/VIP/pull/149/commits/939fdbff5a7f7b16815a01ef44d683234b49517a)